ToolChain/Debugger/Debugger.py

```diff
@@ -21,22 +21,7 @@
 """
 
 from CapuaEnvironment.Capua import Capua
-from Configuration.Configuration import REGISTER_A, \
-                                        REGISTER_B, \
-                                        REGISTER_C, \
-                                        REGISTER_D, \
-                                        REGISTER_E, \
-                                        REGISTER_F, \
-                                        REGISTER_G, \
-                                        REGISTER_A2, \
-                                        REGISTER_B2, \
-                                        REGISTER_C2, \
-                                        REGISTER_D2, \
-                                        REGISTER_E2, \
-                                        REGISTER_F2, \
-                                        REGISTER_G2, \
-                                        REGISTER_S, \
-                                        REGISTER_S2, \
+from Configuration.Configuration import REGISTERS, \
```

Comment:
I don't think that this is too bad here as these are mainly used in single stepping through the debugger... Meaning: there is an expectation that this is going to be slower.

Reply:
But please import from the assembler configuration